onnxruntime/contrib_ops/cuda/inverse.cc

```diff
@@ -150,7 +150,7 @@ Status Inverse::ComputeInternal(OpKernelContext* ctx) const {
   }
 
   IAllocatorUniquePtr<int> info = GetScratchBuffer<int>(num_batches);
-  CUDA_RETURN_IF_ERROR(cudaMemsetAsync(info.get(), 0, num_batches, Stream()));
+  CUDA_RETURN_IF_ERROR(cudaMemsetAsync(info.get(), 0, num_batches * sizeof(int), Stream()));
```

Comment:
Just curious, when will this async op be synced ?

Reply:
It doesn't need syncing right ? When launching operations using a single stream (per-session compute stream : `Stream()`), all CUDA operations in that stream are serialized and so they execute in the order in which they are queued. Anything queued after this on the same stream will execute only after this operation.

Reply:
Yes, Hari is correct.